images/keda/config/main.tf

```diff
@@ -0,0 +1,39 @@
+variable "name" {
+  description = "Component name (e.g. keda, keda-adapter, keda-admission-webhooks)"
+}
+
+variable "package" {
+  description = "Package name (e.g. keda-2.14)"
+}
+
+variable "extra_packages" {
+  description = "Additional packages to install."
+  type        = list(string)
+  default     = []
+}
+
+locals {
+  entrypoints = {
+    "keda"                    = "/usr/bin/keda --zap-log-level=info --zap-encoder=console",
+    "keda-adapter"            = "/usr/bin/keda-adapter --secure-port=6443 --logtostderr=true --v=0",
+    "keda-admission-webhooks" = "/usr/bin/keda-admission-webhooks --zap-log-level=info --zap-encoder=console",
+  }
+}
+
+module "accts" { source = "../../../tflib/accts" }
+
+output "config" {
+  value = jsonencode({
+    contents = {
+      packages = concat([
+        var.package, // keda, keda-adapter, keda-admission-webhooks
+        "busybox",
+        "keda-compat",
```

Comment:
does the version of `keda-compat` ever matter?

Reply:
Not that I can tell, it's just symlinks. The existing images also made use of `keda-compat` which is why I kept it. Happy to change it to be versioned though